### PR TITLE
⬇️ Relax PHP version requirements

### PR DESCRIPTION
**Pull request by n8-dev**
Relax PHP version restrictions

PHP version: `^8.0`  

## Description
Not all our projects are on `8.1+`, many are on `8.0`.

I also reviewed the dependencies in here, they support as low as 7.3

```
phpunit/phpunit 
9.6.13
Requires
  php: >=7.3
```

Also @edwilde is working on something and and needs a wider version support.

### CODE REVIEW

File: composer.json [hunk 1]
```diff
@@ -18,7 +18,7 @@
         "bespoke-phplint"
     ],
     "require": {
-        "php": "^8.1",
+        "php": "^7.3 || ^8.0",
```
Comment (satrun77) [inline composer.json:21]: We should not support PHP 7.x  and 8.0

**Review by satrun77 — CHANGES_REQUESTED** [review]

**Comment by satrun77** [comment]
This change is to be part of version 0.1. Then we are not maintaining an unsupported version of PHP. This PR should not be part of main or version >= 1.x

**Review by satrun77 — APPROVED** [review]
Approved if this is a feature for version 0.1 only

**Comment by n8-dev** [comment]
Cool, I'll pair it back to 8.0.
And make a `0.1` update later if needed.

**Pull request merged**